tests/acceptance/features/apiSpaces/spaceDownload.feature

```diff
@@ -0,0 +1,22 @@
+@api
+Feature: Download space
+  As a user
+  I want to be able to download space
+  So that I can store it in my local storage
+
+
+  Scenario: user downloads a space
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the GraphApi
+    And user "Alice" has uploaded a file inside space "project-space" with content "some data" to "file1.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "other data" to "file2.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "more data" to "file3.txt"
+    When user "Alice" downloads the space "project-space" using the WebDAV API
+    Then the HTTP status code should be "200"
+    And the downloaded "tar" archive should contain these files:
```

Comment:
what about .space file inside a space

Reply:
I didn't see .space folder there.
CC @Salipa-Gurung

Reply:
![Uploading Screenshot from 2023-07-19 16-53-17.png…]()

Reply:
Is that from UI?

Reply:
in js, I think I got the .space file also while trying to implement checking downloaded resource. Maybe this could be limitation of PHP

Reply:
no. while creating new space with API, we also have to explicitly create .space folder